demo/molecule/tabs/playground

```diff
@@ -0,0 +1,198 @@
+)
+
+const items = [
+  {
+    label: 'Tab 1',
+    icon: icon
+  },
+  {
+    label: 'Tab 2',
+    icon: icon
+  },
+  {
+    label: 'Tab 3',
+    icon: icon
+  },
+  {
+    label: 'Tab 4',
+    icon: icon
+  },
+  {
+    label: 'Tab 5',
+    icon: icon
+  },
+  {
+    label: 'Disabled',
+    icon: icon,
+    enabled: false
+  },
+]
+
+
+const itemsNoIcons = [
+  {
+    label: 'Tab 1',
+  },
+  {
+    label: 'Tab 2',
+  },
+  {
+    label: 'Tab 3',
+  },
+  {
+    label: 'Tab 4',
+  },
+  {
+    label: 'Tab 5',
+  },
+  {
+    label: 'Disabled',
+    enabled: false
+  },
+]
+
+function handleClick (index, label) {
+  console.log(index, label)
+}
+
+<h2>Classic Tabs</h2>
+return (
+    <div style={{
+        backgroundColor: 'white',
+        padding: '5px 30px',
+        overflow: 'hidden'
+      }}>
+        <h1>Horizontal Type</h1>
+        <h2>CLASSIC VARIANT</h2>
+        <h3>Default no icons</h3>
+        <MoleculeTabs
+            items={itemsNoIcons}
+            activeTab={0}
+            handleClick={handleClick}
+        />
+        <br/><br/>
+        <h3>Default with icons</h3>
+        <MoleculeTabs
+            items={items}
+            activeTab={0}
+            handleClick={handleClick}
+        />
+        <br/><br/>
+        <h3>Full Width</h3>
+        <MoleculeTabs
+            items={items}
+            activeTab={0}
+            handleClick={handleClick}
+            type={moleculeTabsTypes.FULLWIDTH}
+        />
+        <br/><br/>
+        <h2>HIGHLIGHTED VARIANT</h2>
+        <h3>Default no icons</h3>
+        <MoleculeTabs
+            items={itemsNoIcons}
+            activeTab={0}
+            handleClick={handleClick}
+            variant={moleculeTabsVariants.HIGHLIGHTED}
+        />
+        <br/><br/>
+        <h3>Default with icons</h3>
+        <MoleculeTabs
+            items={items}
+            activeTab={0}
+            handleClick={handleClick}
+            variant={moleculeTabsVariants.HIGHLIGHTED}
+        />
+        <br/><br/>
+        <h3>Full Width</h3>
+        <MoleculeTabs
+            items={items}
+            activeTab={0}
+            handleClick={handleClick}
+            variant={moleculeTabsVariants.HIGHLIGHTED}
+            type={moleculeTabsTypes.FULLWIDTH}
+        />
+        <br/><br/><br/><br />
+        <h1>Vertical Type</h1>
+        
+        <table style={{
+            minWidth: '300px',
+            maxWidth: '50%',
+            float: 'left',
+            marginBottom: '20px'
+          }}>
+          <tr>
+            <td colspan="2">
```

Comment:
as this is jsx, it should be colSpan instead of colspan

Reply:
Sure. My mistake.